day_two_guide/topics/adding_master_host.adoc

```diff
@@ -0,0 +1,177 @@
+=== Adding Masters to *{rhocp}* Environment
+The *{rhocp}* environment can be scaled up horizontally. Masters can be scaled as well regardless if
+`etcd` is collocated on the instances or not. The addition of master instances helps to elevate the load among the OpenShift masters.
+
+NOTE: The current procedures for scaling up masters doesn't scale the etcd
+database if the masters contains etcd in the current environment. For a manual
+procedure on scale etcd, see
+https://docs.openshift.com/container-platform/latest/admin_guide/backup_restore.html#backup-restore-adding-etcd-hosts[adding new etcd hosts]
+documentation.
+
+The scale up playbook performs all the tasks required to add a new master to the current
+*{rhocp}* environment, including generating certificates, installing *{rhocp}* packages and configuring the host itself.
+
+Depending on the installation procedure chosen when deploying the *{rhocp}* environment,
+ there are two different methods for adding nodes to the *{rhocp}* environment:
+
+* Using the quick installer tool
+* Using the advanced install
+
+NOTE: This document focuses on the last option, scale up the environment using the advanced install.
+
+[[requirements]]
+==== Requirements
+This document assumes the instances have been created in the infrastructure. The instances
+must meet the minimum https://docs.openshift.com/container-platform/latest/install_config/install/prerequisites.html#hardware[hardware requirements]
+.
+
+The prerequisites must be completed before running the scale-up procedure
+as the OpenShift installation playbooks do not include the prerequisites tasks, such as register the hosts, installing the
+required packages, `ssh` key exchange for `Ansible`, firewall rules, etc. For
+more information about the prerequisites, see
+https://docs.openshift.com/container-platform/latest/install_config/install/host_preparation.html[host preparation] documentation.
+
+All the tasks are to be performed on the host where the installation was done
+and include the `Ansible` hosts file.
+
+IMPORTANT: If manual changes in the *{rhocp}* environment exist, ensure the
+inventory file reflects those changes prior to the scale up procedure. This includes
+changes to the *{rhocp}* configuration files, for example, modifying the *{rhocp}*
+masters configuration file to customize the *{rhocp}* authentication provider as they may be overwritten.
+
+==== Procedure
+The latest `Ansible` playbooks must be available on the host that is going to do
+the scale up procedure (a workstation or the bastion host) by updating the `atomic-openshift-utils` package:
+
+[subs=+quotes]
+----
+# *yum update atomic-openshift-utils*
+----
+
+*Inventory File*
+
+The `Ansible` hosts file contains the `new_<host type>` group and this group
+belongs to the `OSEv3` group, where `new_<host_type>` can be `new_masters` or `new_nodes`
+
+NOTE: As *{rhocp}* masters are *{rhocp}* nodes, in case adding a new master,
```

Comment:
Missing period and consider rewording:

NOTE: *{rhocp}* masters are *{rhocp}* nodes. When adding a new master, both `new_nodes` and `new_masters` groups must also be added.